src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -976,8 +976,9 @@ services:
       - '@=service("prestashop.core.form.choice_provider.accessible_tab").getChoices()'
       - '@=service("prestashop.core.form.choice_provider.profile").getChoices()'
       - '@=service("prestashop.adapter.multistore_feature").isActive()'
-      - '@=service("prestashop.adapter.employee.avatar_provider").getDefaultAvatarUrl()'
       - "@=service('prestashop.adapter.legacy.configuration')"
+      - '@=service("prestashop.adapter.legacy.configuration").getInt("_PS_ADMIN_PROFILE_")'
```

Comment:
Maybe use the const directly here? (Because using `prestashop.adapter.legacy.configuration` without a shop context is deprecated)
```suggestion
      - !php/const _PS_ADMIN_PROFILE_
```

Reply:
Using `!php/const` triggers a new problem: all constants are `string` and can't be casted in `integer`.

May be we should create in the future a service for fetching constants and cast them ?

Reply:
![Screenshot from 2022-09-16 13-00-53](https://user-images.githubusercontent.com/31609858/190614037-62eaea05-99d0-4e49-a65c-3e25fb6588aa.png)
seems to work fine even with strict types declaration

Reply:
Fixed